ext/src/test/scala/com/quantifind/sumac/scalatra/SumacSupportTest.scala

```diff
@@ -100,14 +100,16 @@ class SumacSupportTest extends ScalatraFunSuite {
   }
 
   def verifyParams(path: String, parameters: JArray) {
-    val names = parameters.arr.map{v => (v \ "name").asString}.toSet
+    // TODO figure out what has changed here -- the swagger spec itself?  just some json4s detail?
```

Comment:
Could you add a bit more context here about the failure you were seeing?  While it probably isn't a big deal, that would help a lot when deciding how safe a change this is (as well as help when someone comes back to address the TODO.)

Reply:
sure, I didn't want to get bogged down if this PR wasn't going to go anywhere, but yeah I can add some detail here.  I think there was an extra level of nesting in the generated json now.  I assume this is because I bumped the version of scalatra, and it's decided to generate something a little different than before.

Reply:
:+1: